src/components/AppHeader.module.css

```
@@ -186,3 +186,6 @@
 .app-container[data-layout-tier="tablet"] .app-header-brand-wordmark .fretflow-wordmark {
   height: clamp(1.35rem, 2.65vw, 2.05rem);
 }
+dmark :global(.fretflow-wordmark) {
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)